File: JOSÉ_ALEJANDRO_GÓMEZ_PAREDES_TLL4_ 03.py
# escribir archivo desde o en memoria
import io

Fl_Stream = io.StringIO()


Fl_Stream.write("Esta es la primera liena de mi primer archivo"), Fl_Stream.write(
    "Esta es la segunda liena de mi primer archivo\n"
), Fl_Stream.write("Esta es la tercera liena de mi primer archivo con salto de linea"),

## recuperamos la información de la memoria

Lv_información = Fl_Stream.getvalue()

## tranferir informacion a un archivo fisico

with open("Archivo170523_03.txt", "w") as archivo_disco:
    archivo_disco.write(Lv_información)

print("su tercer archivo ha sido creado, vamos bien")

## todo se crea en memoria y luego se guarda en un archivo fisico

File: talleres en js/JOSÉ_ALEJANDRO_GÓMEZ_PAREDES_TLL4_03.js
const fs = require("fs");
const Fl_Stream =
  "Esta es la primera línea de mi primer archivo\nEsta es la segunda línea de mi primer archivo\nEsta es la tercera línea de mi primer archivo con salto de línea";
const Lv_información = Fl_Stream;
const archivo_disco = fs.createWriteStream("Archivo170523_03.txt");
archivo_disco.write(Lv_información);
console.log("Su tercer archivo ha sido creado, vamos bien!");
